build-android-start/app/src/main/java/com/google/firebase/codelab/friendlychat/MainActivity.kt

```diff
@@ -44,6 +46,12 @@ class MainActivity : AppCompatActivity() {
         // Initialize Firebase Auth and check if the user is signed in
         // TODO: implement
 
+        val gso = GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_SIGN_IN)
```

Comment:
Is this edit in the `start` folder intentional?

Reply:
Yes. I shouldn't have removed it from the start folder - it's in the final code